### DISCUSSION TITLE
Prisma Client Dart v4.0.0-beta.3

### DISCUSSION THREAD

**Discussion by medz** [medz/dart-orm#330] [Announcements]
To install Prisma Client Dart v4.0.0-beta.3 run:

```bash
dart pub add orm:4.0.0-beta.3
```

Or update your `pubspec.yaml` file:

```yaml
dependencies:
  orm: 4.0.0-beta.3
```

## What's Changed

- **Bug**: Fixed list of enum not parsing correctly (https://github.com/medz/prisma-dart/issues/329)

## Credits

- @lucasvaudey
- @NeroSong

<hr /><em>This discussion was created from the release <a href='https://github.com/medz/prisma-dart/releases/tag/4.0.0-beta.3'>Prisma Client Dart v4.0.0-beta.3</a>.</em>

**Comment by kidusdev** [comment]
When will the stable version be released? Any announcements

**Reply by medz** [reply]
I am also further confirming whether there are still serious bugs. Because this change has almost rewritten the whole ORM.

**Comment by AndryHTC** [comment]
https://github.com/medz/prisma-dart/issues/289#issuecomment-1956755708
> @medz how can I get started to migrate to the [4.0.0](https://github.com/medz/prisma-dart/releases/latest), so I can help in testing it?
> 
> I've tried the migration instruction from the [alpha-0](https://github.com/medz/prisma-dart/discussions/292) but I had no joy as the generation does not even start
